x-pack/test/alerting_api_integration/spaces_only/tests/alerting/event_log_alerts.ts

```diff
@@ -110,7 +110,7 @@ export default function eventLogAlertTests({ getService }: FtrProviderContext) {
           case 'active-instance':
             expect(instanceEvents[i]?.kibana?.alerting?.instance_id).to.equal('instance');
             expect(instanceEvents[i]?.kibana?.alert?.flapping).to.equal(false);
-            expect(instanceEvents[i]?.event?.start).to.equal(currentAlertSpan.start);
+            expect(instanceEvents[i]?.event?.start).not.to.equal(undefined);
```

Comment:
I don't understand these changes.  I don't think we want to be changing `currentAlertSpan.alertId` or `.start` in `active-instance` - the idea is to track what we get from new-instance, in the subsequence *-instance events, to make sure they make sense.  For instance, instead of reseting `currentAlertSpan.alertId`, we should be testing it against the specified `instanceEvents[i]`.  Same with `start` I think.

Reply:
Resolved in https://github.com/elastic/kibana/pull/144415/commits/bd748cd1fa3fdbcb3cdd8689b593a9088afd6756